tests/lib/rules/id-length.js

```diff
@@ -35,6 +35,7 @@ ruleTester.run("id-length", rule, {
         "var xyz = new ΣΣ();",
         "unrelatedExpressionThatNeedsToBeIgnored();",
         "var obj = { 'a': 1, bc: 2 }; obj.tk = obj.a;",
+        "var query = location.query.q || '';",
```

Comment:
It does seem valuable as a regression test to me, since we're trying to ensure that both this case and ternaries are treated similarly